[DOCS-3161] Add Watchdog RCA FAQ (#13188)
DataDog/documentation: content/en/watchdog/faq/root-cause-not-showing.md

---
title: Root Cause Not Showing
kind: faq
further_reading:
- link: "/watchdog/#root-cause-analysis-for-apm-beta"
  tag: "Documentation"
  text: "Watchdog Root Cause Analysis for APM"
---

Watchdog looks for specific types of root causes. We're working on adding additional coverage. Currently, Watchdog Root Cause Analysis (RCA) detects the following root causes:

- Version changes, as captured by the `version` tag on your [APM services][1]
- Traffic increases, as captured by hit rate metrics on your APM-instrumented services
- AWS instance failures, as captured by [AWS EC2 integration metrics][2]
- Running out of disk space, as captured by [system metrics][3] from the Datadog agent

If you don't see a root cause, it's likely that the specific root cause is not one of the types described above or there isn't instrumentation configured to capture it.

Watchdog Root Cause Analysis works best when:

- You use distributed tracing, so that Watchdog knows the dependency structure between your services
- You use [unified service tagging][4], so that Watchdog knows when you deploy new code and can link infrastructure and APM metrics

> Do you think Watchdog should have found something that it missed? Or, do you have another type of root cause we should prioritize adding? We're always looking to make Watchdog smarter, so [open a support request][5] to let us know.

Reviewer: ```suggestion
Do you think Watchdog should have found something that it missed? Or, do you have another type of root cause that you would like to be added? Datadog is always looking to make Watchdog smarter, so [open a support request][5] to let us know.
```
